source/_integrations/reolink.markdown

```diff
@@ -194,6 +195,8 @@ Depending on the supported features of the camera, switch entities are added for
 - Buzzer on event
 - Email on event
 - FTP upload
+- PIR enabled*
+- PIR reduce false alarm*
```

Comment:
Are there any descriptions to go with the *ed entries?

I'm not sure what the * means.

Reply:
Its at the top of the documentation: https://www.home-assistant.io/integrations/reolink/#asterisk--next-to-entities-listed-in-this-documentation